pyanaconda/modules/runtime/user_interface/ui.py

```diff
@@ -113,6 +124,19 @@ def set_display_mode_non_interactive(self, non_interactive):
         self.display_mode_nonInteractive_changed.emit()
         log.debug("Display mode non-interactive set to: %s", str(non_interactive))
 
+    @property
+    def display_mode_text_kickstarted(self):
+        """Report if text mode was explicitely requested via kickstart.
+
+
+        #NOTE: No setter as this is only set once when parsing the kickstasrt.
```

Comment:
What about `int.text`?

Reply:
The current behavior is like this:

- `inst.text` - ask if you really want TUI or VNC
- `text` in kickstart - go directly to TUI
- nothing - boot to GUI

As this property talks about text mode being set via kickstart, I think it is still correct. :)